mama/c_cpp/src/c/datetimeimpl.h

```diff
@@ -26,70 +26,82 @@
 extern "C" {
 #endif
 
-typedef uint64_t mama_time_t;
+typedef struct mama_time_t_
+{
+    mama_i64_t              mSeconds;
```

Comment:
Do we have to hide the documented implementation details behind MAMA typedefs?
I've prototyped this struct as:
    int64_t tv_sec;
    long int tv_nsec;
    uint8_t tv_presision;
    uint8_t tv_hints;
IMO it's much more clear and hints that data fields are mimicked after  timeval.